src/front-end/typescript/lib/pages/organization/edit/tab/team.tsx

```diff
@@ -160,6 +165,32 @@ const update: Update<State, Msg> = ({ state, msg }) => {
       ];
     }
 
+    case 'approveAffiliation':
+      return [
+        state
+          .set('approveAffiliationLoading', msg.value.id)
+          .set('showModal', null),
+        async (state, dispatch) => {
+          state = state.set('approveAffiliationLoading', null);
+          const result = await api.affiliations.update(msg.value.id, null);
```

Comment:
For Brianna's learning: Why does update take `null` as an argument?

Reply:
This setting the approveAffiliationLoading to 'null'.  I believe when that variable is truthy, the buttons are greyed out, when it's falsy the buttons are active.